var/spack/repos/builtin/packages/laghos/package.py

```diff
@@ -41,7 +41,7 @@ class Laghos(MakefilePackage):
     version('develop', git=git, branch='master')
 
     depends_on('mpi')
-    depends_on('mfem@laghos-v1.0', when='@1.0')
+    depends_on('mfem@laghos-v1.0:', when='@1.0')
```

Comment:
I think version ranges do not work with versions that begin with letters. In particular `laghos-v1.0` is ordered before any numbered version. The only exception is `develop` which is always larger than any other version string. In this case I think we can use `depends_on('mfem@laghos-v1.0,3.3.2:', when='@1.0')`.

Reply:
Yeah I wasn't sure how the version strings ranges worked on git repos.  I'll give the comma separated list a try.